libraries/Aws/ServiceHealth/service_health.py

```diff
@@ -0,0 +1,88 @@
+from itertools import chain, product
+from datetime import datetime
+import requests
+import yaml
+from RW.Utils import parse_timedelta, csv_to_list
+
+GLOBAL_SERVICES = [  #List from https://d3s31nlw3sm5l8.cloudfront.net/services.json
+    "supportcenter",
+    "marketplace",
+    "chime",
+    "resourcegroups",
+    "organizations",
+    "iam",
+    "import-export",
+    "interregionvpcpeering",
+    "trustedadvisor",
+    "route53domainregistration",
+    "spencer",
+    "account",
+    "management-console",
+    "billingconsole",
+    "route53apprecoverycontroller",
+    "cloudfront",
+    "globalaccelerator",
+    "awswaf",
+    "health",
+    "apipricing",
+    "route53",
+    "chatbot",
+]
+
+
+class ServiceHealth:
+    def get_event_json(self) -> dict:
+        url = "https://history-events-eu-west-1-prod.s3.amazonaws.com/historyevents.json"
```

Comment:
may want to pass in the configured region eg `eu-west-1` so users have control over the pulled events

Reply:
Good point, I fully agree with you, but the reason behind the region being hard-coded is the fact, that not every region contains such a bucket. For instance, you may use `us-east-1` (actually this is the better one to hardcode) but if you try to use `us-west-1` or some other random one, it will fail. Leaving it a field to be filled by the end-user may lead to mistakes. What would be an efficient and easy way to give this piece of information to a hypothetical user? I assume that the goal is to make codebundles as easy to use as possible, so maybe a list of predefined choices. But still, list of predefined elements is something that needs to be added in the application. So maybe, how about adding a default value and an explicit description on which particular regions are allowed, wdyt?

Reply:
and to be clear, the region here is only for the bucket location and does not impact the `historyevents.json` content which is supposed to be identical in every location

Reply:
This is a great question. For many of our codebundles we've gone back and forth on configuration style. A few times we've written really powerful codebundles but the configuration was 12+ fields and some of those were optional. We found this to be really confusing to end users or during tutorial sessions.

Ideally I try to keep config to < 8 fields, so in this case I think it's fair to add another field with a drop down list for the user to select from. You can do that like so:

```
    RW.Core.Import User Variable    MY_DROPDOWN
    ...    type=string
    ...    description=a description of the dropdown
    ...    pattern=\w*
    ...    enum=[Value1, Value2, Etc] 
    ...    default=Value2
```
eg:
https://github.com/runwhen-contrib/rw-public-codecollection/blob/main/codebundles/prometheus-queryinstant-transform/sli.robot#L52

Of course you could try write functionality that detects the bucket region in a "smart" way geographically but I think that's a tad complex for a first revision of this codebundle. I'd defnitely consider that out of scope.

Reply:
the dropdown sound like a perfect solution, thank you

Reply:
Dropdown added along with the list of supported regions